src/runtime/pkg/katautils/config.go

```diff
@@ -306,6 +311,20 @@ func (h hypervisor) hotPlugVFIO() config.PCIePort {
 	return h.HotPlugVFIO
 }
 
+func (h hypervisor) pcieRootPort() uint32 {
+	if h.PCIeRootPort > maxPCIeRootPorts {
+		return maxPCIeRootPorts
+	}
+	return h.PCIeRootPort
+}
+
```

Comment:
We only need to check for max values since a negative value is catched by the TOML parsing logic. If it exceeds the max value we need to truncate.